cluster/saltbase/salt/kube-node-unpacker/kube-node-unpacker.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
I would prefer not to spend time optimizing the on node unpacking since we are going to rip it out imminently. I'm working on #8686 now and it'll be done before this code makes it into a release revision.